src/server/metadata/cmds/cmds.go

```diff
@@ -37,3 +46,82 @@ func Cmds(mainCtx context.Context, pachctlCfg *pachctl.Config) []*cobra.Command
 
 	return commands
 }
+
+func parseKV(data string) (k, v string) {
+	// TODO(jrockway): Allow keys and values with = in them.
```

Comment:
How would the user specify a key of `foo=bar` and a value of `baz=quux`? Perhaps `foo\=bar=baz\=quux` (all non-separating equal signs escaped), or `foo\=bar=baz=quux` (only first unescaped equal sign separates)?

Or do you just mandate the JSON format for keys or values containing `=`? May be worth determining now, rather than putting off — or at least adding a story and referring to it in the `TODO(QQQ)` line.

Reply:
Yes.  The current format is totally unsuitable for production use.  I opened CORE-2228 to design this; there will be a short design doc on what we want here.